lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -699,20 +704,17 @@ WasmBytecodeGenerator::EmitLoop()
     Js::ByteCodeLabel loopLandingPadLabel = m_writer.DefineLabel();
 
     uint loopId = m_writer.EnterLoop(loopHeadLabel);
-    PushLabel(loopTailLabel);
```

Comment:
Yes, tail label doesnt exist anymore, you need an enclosing block to simulate a break out of the loop